Apps/W1/HybridGP/app/src/Migration/Support/GPPaymentTerms.Page.al

```diff
@@ -0,0 +1,113 @@
+namespace Microsoft.DataMigration.GP;
+
+page 40134 "GP Payment Terms"
+{
+    ApplicationArea = All;
+    Caption = 'GP Payment Terms';
+    PageType = Worksheet;
+    SourceTable = "GP Payment Terms";
+    UsageCategory = Lists;
+    InsertAllowed = false;
+    DeleteAllowed = false;
+    Editable = true;
+
+    layout
+    {
+        area(content)
+        {
+            repeater(General)
+            {
+                field(DateFormulaIsValid; DateFormulaIsValid)
+                {
+                    Caption = 'Is Valid?';
```

Comment:
We do not have ? in the captions. Valid or Valid Date Formula is better

Reply:
Changed to Valid Date Formula